index.html

```diff
@@ -199,9 +199,39 @@ <h3 class="promo-box__title font-secondary">Point Break</h3>
 
         </article>
 
+    </section>
+
+    <section class="section-join">
+        <h3 class="join__title font-secondary">Join the Club</h3>
+        <p class="join__text font-tertiary">
+            By better understanding the various aspects of surfing, you will improve faster and have more fun in the water.
+        </p>
+
+        <form>
+            <input type="email" name="e-mail" placeholder="your e-mail" required >
+            <button type="submit">
+                <svg class="submit-button" xmlns="http://www.w3.org/2000/svg" width="10" height="18" viewBox="0 0 10 18">
+                    <path fill="#fff" d="M9.85 8.64L.92.15a.55.55 0 0 0-.75 0 .49.49 0 0 0 0 .72L8.7 9 .17 17.13a.49.49 0 0 0 0 .72.54.54 0 0 0 .75 0l8.92-8.49a.5.5 0 0 0 0-.72z"/>
+            </svg>
+        </button>
+        </form>
```

Comment:
1. Make a better 'focused' effect
2. Make better text alignment, when writing e-mail
3. Make better :hover effect for the submit button